packages/terra-form-input/src/Input.module.scss

```diff
@@ -53,24 +53,24 @@
 
     &:focus {
       background-color: var(--terra-form-input-focus-background-color, #fff);
-      background-size: var(--terra-form-input-focus-keyboard-animation-size);
+      background-size: var(--terra-form-input-focus-keyboard-animation-background-size);
       border: var(--terra-form-input-focus-border, 1px solid #4cb2e9);
       color: var(--terra-form-input-focus-color);
       outline: none;
 
       [dir=ltr] & {
-        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-ltr);
+        box-shadow: var(--terra-form-input-focus-keyboard-ltr-box-shadow);
       }
 
       [dir=rtl] & {
-        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-rtl);
+        box-shadow: var(--terra-form-input-focus-keyboard-rtl-box-shadow);
       }
     }
   }
 
   .form-error {
     background-color: var(--terra-form-input-error-background-color, #fff);
-    background-image: var(--terra-form-input-error-focus-animation-background);
+    background-image: var(--terra-form-input-error-focus-animation-background-image);
```

Comment:
nit: I'd drop `focus-animation` from this vairable name

Reply:
https://github.com/cerner/terra-core/pull/2125/commits/d89bf406444dc93fa1237dae7fc382f0ec92b6b0#diff-462afbdb742ea35bac7504edbfd87d7dR73